### PR TITLE
added filters on the runcard

### PR DESCRIPTION
**Pull request by jordivallsq**
Created filters in the controlers section of QUA config

### CODE REVIEW

File: docs/releases/changelog-dev.md [hunk 1]
```diff
@@ -57,6 +57,27 @@
 
   [#762](https://github.com/qilimanjaro-tech/qililab/pull/762)
 
+- Added `filter` argument inside the qua config file compilation from runcards with qm clusters. This is an optional element for distorsion filters that includes feedforward and feedback, two distorion lists for distorsion compensation and fields in qua config filter. These filters are calibrated and then introduced as compensation for the distorsions of the pulses from external sources such as Bias T. The runcard now might include the new filters (optional):
```
Comment (GuillermoAbadLopez) [inline docs/releases/changelog-dev.md:60]: Again, great detailed changelog! Thanks!

File: docs/releases/changelog-dev.md [hunk 2]
```diff
@@ -57,6 +57,27 @@
...
+
+  ```
```
Comment (GuillermoAbadLopez) [inline docs/releases/changelog-dev.md:62]: I liked this Example thing here oyu added on the other PR, maybe add it here aswell?
```suggestion
  Example:
    ```
```
Notice you will need to tab the rest of the Example too!

File: src/qililab/instruments/quantum_machines/quantum_machines_cluster.py
Comment (GuillermoAbadLopez) [inline src/qililab/instruments/quantum_machines/quantum_machines_cluster.py:1]: Maybe like in the other PR add these new filters to the `data.py`, 
Or make sure this `dict` structure is checked somewhere in the test? 👀

**Review by GuillermoAbadLopez — CHANGES_REQUESTED** [review]
Look really nice! GJ

Left 2 silly comments!

File: src/qililab/instruments/quantum_machines/quantum_machines_cluster.py
Comment (jordivallsq) [reply src/qililab/instruments/quantum_machines/quantum_machines_cluster.py:1]: Added those on data for each type of runcard

**Comment by jordivallsq** [comment]
Adressed all the comments

File: docs/releases/changelog-dev.md [hunk 3]
```diff
@@ -57,6 +57,28 @@
 
   [#762](https://github.com/qilimanjaro-tech/qililab/pull/762)
 
+- Added `filter` argument inside the qua config file compilation from runcards with qm clusters. This is an optional element for distorsion filters that includes feedforward and feedback, two distorion lists for distorsion compensation and fields in qua config filter. These filters are calibrated and then introduced as compensation for the distorsions of the pulses from external sources such as Bias T. The runcard now might include the new filters (optional):
+
+  Example:
+    ```
+  instruments:
```
Comment (GuillermoAbadLopez) [inline docs/releases/changelog-dev.md:64]: Watch out, all the example needs to be **indented**!

Not only the line I added in the commit, sorry for the confusion!

**Review by GuillermoAbadLopez — APPROVED** [review]
GJ!

Just one last thing to adress before testing on hardware and merge! 🙌

File: docs/releases/changelog-dev.md (hunk 3 above)
Comment (jordivallsq) [reply docs/releases/changelog-dev.md]: Done!

**Comment by jordivallsq** [comment]
The filters have been tested in QM OPX and the results look the same as the ones measured by Mireia.

Before filters:
imagen

After filters:
imagen

**Pull request merged**